backend/src/main/scala/com/softwaremill/bootzooka/infrastructure/Doobie.scala

```diff
@@ -31,8 +29,8 @@ object Doobie
   implicit def taggedStringMeta[U]: Meta[String @@ U] =
     implicitly[Meta[String]].asInstanceOf[Meta[String @@ U]]
 
-  implicit val passwordHashMeta: Meta[PasswordHash[SCrypt]] =
-    implicitly[Meta[String]].asInstanceOf[Meta[PasswordHash[SCrypt]]]
+  implicit val passwordHashMeta: Meta[String @@ HashedPassword] =
```

Comment:
maybe simply `String @@ Hashed`? The fact that it's a password doesn't add much information in this context?

Reply:
👍